draft-ietf-tls-ticketrequests.md

```diff
@@ -105,10 +105,10 @@ as shown here.
 
 The ability to request one or more tickets is useful for a variety of purposes:
 
-- Parallel HTTP connections: To minimize ticket reuse while still improving performance, it may
-be useful to use multiple, distinct tickets when opening parallel connections. Clients must
-therefore bound the number of parallel connections they initiate by the number of tickets
-in their possession, or risk ticket re-use.
+- Parallel HTTP connections: To improve performance, a client may open parallel connections.
+To avoid ticket reuse, the client may use multiple, distinct tickets on each connection.
```

Comment:
Hmm, is "multiple" still correct here?
Having distinct tickets on each connection causes the client to use multiple tickets overall, but this looks like it is parsing as saying that there are multiple tickets per connection, which doesn't seem right.

Reply:
I'll fix directly on master.

Reply:
(Fixed)